auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -1,90 +1,102 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
OK, but you still don't need coordinatorlayout, you don't have any FAB or toolbar. Simple FrameLayout is enough.